### PR TITLE
Feat i18n

### PR DESCRIPTION
**Pull request by mixcas**

### CODE REVIEW

File: app/imports/i18n/es.i18n.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "apply": {
```
Comment (varffs) [inline app/imports/i18n/es.i18n.json]: why is this duplicated?

File: app/imports/startup/i18n.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+import 'import/i18n/en.js';
+
+function getLang() {
+  if (navigator.languages != undefined)  {
+    return navigator.languages[0].substring(0, 2);
+  }
+  return navigator.language || navigator.browserLanguage;
+}
+
+i18n.setLocale(getLang());
```
Comment (varffs) [inline app/imports/startup/i18n.js]: this will not allow users to explicitly set a language, but perhaps we do a check on login?

Comment (varffs) [inline app/imports/startup/i18n.js]: i mean a lang preference

Comment (varffs) [inline app/imports/startup/i18n.js]: also this is the server code isnt it? why bother checking a navigator in this context anyway?

File: app/imports/i18n/es.i18n.json (hunk 1 above)
Comment (mixcas) [inline app/imports/i18n/es.i18n.json]: oh, my bad

**Comment by varffs** [comment]
I guess there is only supposed to be 1 local config file per local so once that is clear everything else seems fine.

but there are 8 linting problems to fix too

**Comment by varffs** [comment]
@mixcas you just need to sort out the duplicated files and this can be merged

**Comment by mixcas** [comment]
This is ready. I didn't solve all of the linting cuz i think i already fixed those in a diff branch.

**Pull request merged**